extensions/amp-form/0.1/amp-form.js

```diff
@@ -169,8 +142,14 @@ export class AmpForm {
     this.form_.addEventListener('submit', e => this.handleSubmit_(e), true);
     const inputs = this.form_.querySelectorAll('input,select,textarea');
     for (let i = 0; i < inputs.length; i++) {
```

Comment:
I meant actually to do that but totally forgot about it. Thanks for the catch. Done.